docs/en/tutorials/test_results_submission.md

```diff
@@ -0,0 +1,65 @@
+# Tutorial 11: Test Results Submission
+
+## Panoptic segmentation test results submission
+
+The following sections introduce how to produce the prediction results of panoptic segmentation models on the COCO test-dev set and submit the predictions to [COCO's evaluation server](https://competitions.codalab.org/competitions/19507).
+
+### Prerequisites
+
+- Download [COCO test dataset images](http://images.cocodataset.org/zips/test2017.zip), [testing image info](http://images.cocodataset.org/annotations/image_info_test2017.zip), and [panoptic train/val annotations](http://images.cocodataset.org/annotations/panoptic_annotations_trainval2017.zip), then unzip them, put 'test2017' to `data/coco/`, put 'panoptic_val2017.json', 'image_info_test-dev2017.json' and 'image_info_test2017.json' to `data/coco/annotations/`.
+
+- Run following code to update category information in testing image info. Since the attribute `isthing` is missing in category information of 'image_info_test-dev2017.json', we need to update it with the category information in 'panoptic_val2017.json'.
+
+```shell
+python tools/misc/update_coco_panoptic_test_image_info.py data/coco/annotations
+```
+
+- A checkopint and a config file of panoptic model in mmdetection. Such as the [config file](https://download.openmmlab.com/mmdetection/v2.0/maskformer/maskformer_r50_mstrain_16x1_75e_coco/maskformer_r50_mstrain_16x1_75e_coco.py) of maskformer_r50 and its [checkpoint](https://download.openmmlab.com/mmdetection/v2.0/maskformer/maskformer_r50_mstrain_16x1_75e_coco/maskformer_r50_mstrain_16x1_75e_coco_20220221_141956-bc2699cb.pth).
```

Comment:
This should not be in the Prerequisites. It should be put in the examples. You can say: `suppose you have a config and its checkpoint, e.g., ...`